templates/skeleton/app/routes/[.well-known].$.tsx

```diff
@@ -0,0 +1,19 @@
+import type {ActionFunctionArgs, LoaderFunctionArgs} from '@shopify/remix-oxygen';
+
+export async function loader({request}: LoaderFunctionArgs) {
+  throw new Response(`${new URL(request.url).pathname} not found`, {
+    status: 404,
+  });
+}
+
+export async function action({request}: ActionFunctionArgs) {
+  const {pathname} = new URL(request.url);
+
+  if (pathname === '/.well-known/shopify/monorail/unstable/produce_batch') {
```

Comment:
Why is this necessary?

Reply:
bundled in the WebPixel - couldn't override it to use the global endpoint

Reply:
Not sure if this is useful, but we have a proxy for .well-known in the Oxygen worker which redirects to the assets server: https://github.com/Shopify/hydrogen/blob/main/packages/cli/src/lib/mini-oxygen/workerd.ts#L218